spec/beaker-pe/install/pe_utils_spec.rb

```diff
@@ -444,6 +444,79 @@ def slice_installer_options(host)
     end
   end
 
+  describe 'add_extended_gpg_key_to_hosts' do
+    let(:on_cmd) { 'curl http://apt.puppetlabs.com/pubkey.gpg | apt-key add -' }
+    let(:host) do
```

Comment:
Maybe it should be called out that this is the _debian_ host? `debian_host` or something? The specs below are a little confusing to read when they just say `host`, my mind immediately thinks "yes, but which host?"